README.md

```diff
@@ -1494,6 +1528,8 @@ In general the configuration parameters are the same as the command line argumen
 | ft_group | In Fault Tolerance mode, you can start a group of streaming servers with only one server being active while others are running in standby mode. This is the name of this FT group | String | `ft_group: "my_ft_group"` |
 | partitioning | If set to true, a list of channels must be defined in store_limits/channels section. This section then serves two purposes, overriding limits for a given channel or adding it to the partition | `true` or `false` | `partitioning: true` |
 | cluster | Cluster Configuration | Map: `cluster: { ... }` | **See details below** |
+| encrypt | Specify if server should encrypt messages (only the payload) when storing them | `true` or `false` | `encrypt: true` |
```

Comment:
We don't talk anywhere about key rotation.

Reply:
Key is not rotated. The nonce is generated after a random number of time it was used for encryption. What do you have in mind?

Reply:
We can solve that later..